### ISSUE TITLE
PostmarkError: Request body larger than maxBodyLength limit

### ISSUE THREAD

**Issue by dcts** [wildbit/postmark.js#75]
When I try to send a batch of emails with `sendEmailBatch()` and one of the emails has a large body, then the sending of the whole batch fails and I get this error:
```
PostmarkError: Request body larger than maxBodyLength limit
```

This seems to be a known issue, it was discussed on [StackOverflow](https://stackoverflow.com/questions/56868023/error-request-body-larger-than-maxbodylength-limit-when-sending-base64-post-req). In my opinion this is a bug and can be fixed by setting `maxBodyLength` to `Infinity`. I did a [pull request](https://github.com/wildbit/postmark.js/pull/74) that fixes this problem. Let me know what you think of this. 

Are there any downsides of adapting the `maxBodyLength` to `Infinity`?

**Comment by ibalosh** [comment]
hey @dcts , 

this update was released in latest version 2.7.1

**Comment by dcts** [comment]
Awesome thanks a lot!
